apps/notifications/src/lib/zod-form-ast/zod-form-ast.ts

```diff
@@ -0,0 +1,139 @@
+import type { ZodTypeAny } from 'zod';
+import { ZodFirstPartyTypeKind } from 'zod';
+
+export function toAST(schema: ZodTypeAny): Record<string, any> {
```

Comment:
question (non-blocking): What does this function do?

Reply:
oh, forgot to remove it. This is to convert a zod schema to a structure which can be used to dynamically build forms in ui. This one is working as expected but i'd need to actually try building a form to verify

Reply:
adding it in here just in case https://gist.github.com/ygrishajev/d8c180ddcc3fb52710460fe62238656b